package.json

```diff
@@ -8,52 +8,53 @@
     "@braintree/sanitize-url": "^7.0.0",
     "@floating-ui/dom": "^1.4.5",
     "@phosphor-icons/vue": "^2.2.1",
-    "@tiptap/core": "^2.0.3",
-    "@tiptap/extension-bold": "^2.0.3",
-    "@tiptap/extension-bullet-list": "^2.0.3",
-    "@tiptap/extension-document": "^2.0.3",
-    "@tiptap/extension-hard-break": "^2.0.3",
-    "@tiptap/extension-heading": "^2.0.3",
-    "@tiptap/extension-history": "^2.0.3",
-    "@tiptap/extension-italic": "^2.0.3",
-    "@tiptap/extension-link": "^2.0.3",
-    "@tiptap/extension-list-item": "^2.0.3",
-    "@tiptap/extension-mention": "^2.0.3",
-    "@tiptap/extension-ordered-list": "^2.0.3",
-    "@tiptap/extension-paragraph": "^2.0.3",
-    "@tiptap/extension-strike": "^2.0.3",
-    "@tiptap/extension-table": "^2.0.3",
-    "@tiptap/extension-table-cell": "^2.0.3",
-    "@tiptap/extension-table-header": "^2.0.3",
-    "@tiptap/extension-table-row": "^2.0.3",
-    "@tiptap/extension-text": "^2.0.3",
-    "@tiptap/extension-underline": "^2.0.3",
-    "@tiptap/pm": "^2.0.3",
-    "@tiptap/suggestion": "^2.0.3",
-    "@tiptap/vue-3": "^2.1.12",
+    "@tiptap/core": "^3.0.0",
+    "@tiptap/extension-bold": "^3.0.0",
+    "@tiptap/extension-bullet-list": "^3.0.0",
+    "@tiptap/extension-document": "^3.0.0",
+    "@tiptap/extension-hard-break": "^3.0.0",
+    "@tiptap/extension-heading": "^3.0.0",
+    "@tiptap/extension-history": "^3.0.0",
+    "@tiptap/extension-italic": "^3.0.0",
+    "@tiptap/extension-link": "^3.0.0",
+    "@tiptap/extension-list-item": "^3.0.0",
+    "@tiptap/extension-mention": "^3.0.0",
+    "@tiptap/extension-ordered-list": "^3.0.0",
+    "@tiptap/extension-paragraph": "^3.0.0",
+    "@tiptap/extension-strike": "^3.0.0",
+    "@tiptap/extension-table": "^3.0.0",
+    "@tiptap/extension-table-cell": "^3.0.0",
+    "@tiptap/extension-table-header": "^3.0.0",
+    "@tiptap/extension-table-row": "^3.0.0",
+    "@tiptap/extension-text": "^3.0.0",
+    "@tiptap/extension-underline": "^3.0.0",
+    "@tiptap/pm": "^3.0.0",
+    "@tiptap/suggestion": "^3.0.0",
+    "@tiptap/vue-3": "^3.0.0",
     "@uppy/core": "^3.0.1",
     "@uppy/drag-drop": "^3.0.0",
     "@uppy/progress-bar": "^3.0.0",
     "@uppy/tus": "^3.0.1",
     "@vue/server-renderer": "^3.4.29",
     "@vue/vue3-jest": "^29.2.6",
+    "@vueuse/integrations": "^11.1.0",
     "a11y-datepicker": " ^0.9.0",
     "dayjs": "^1.11.5",
     "dompurify": "^3.0.0",
     "fscreen": "^1.2.0",
     "ismobilejs": "^1.1.1",
     "lscache": "^1.3.2",
     "plyr": "^3.7.2",
-    "prosemirror-model": "^1.21.1",
+    "prosemirror-model": "1.22.3",
```

Comment:
As a first step to clean up the versioning conflict mess, i'd suggest to get rid of direct access to prosemirror  by [changing this line](https://github.com/demos-europe/demosplan-ui/blob/e624fc6fc892dfe366d8b275bb9bc19a017f68fb/src/components/DpEditor/libs/editorInsertAtCursorPos.js#L1) to 

```
import { DOMParser } from '@tiptap/pm/model'
```